### PR TITLE
Organization balance report

### PR DESCRIPTION
**Pull request by garyhtou**
https://bank--pr-2433.herokuapp.com/admin/balances

image

image

also added `JSON` export cause it was easy

### CODE REVIEW

**Comment by garyhtou** [comment]
fyi @exu3, this PR will most likely cause merge conflicts with https://github.com/hackclub/bank/pull/2406

**Comment by maxwofford** [comment]
This looks great & addresses the issue! Go ahead and merge this once you're ready!

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -1012,9 +1012,8 @@ def balances
       [:end_date, ->(_) { @end_date }]
     ]
     serializer = ->(event) do
-      template.reduce({}) do |hash, (header, field)|
+      template.each_with_object({}) do |(header, field), hash|
```
Comment (garyhtou) [inline app/controllers/admin_controller.rb]: dang, cool method

Comment (garyhtou) [reply app/controllers/admin_controller.rb]: makes it so that you don't need to return the hash on every loop

**Pull request merged**